cherrypy/tutorial/Dockerfile

```diff
@@ -5,12 +5,12 @@ COPY . /tutorial
 WORKDIR /tutorial
 
 RUN pip install cherrypy && \
-  sed -i 's/^server\.socket_host.*$/server.\socket_host = "0.0.0.0"/' \
+  sed -i 's/^server\.socket_host.*$/server\.socket_host = "0.0.0.0"/' \
```

Comment:
I'm not sure whether the replacement part should be escaped.

Reply:
Indeed you are right.